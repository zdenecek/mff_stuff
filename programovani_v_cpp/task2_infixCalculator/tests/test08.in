4
a=5
a=6
b=$a
$b
