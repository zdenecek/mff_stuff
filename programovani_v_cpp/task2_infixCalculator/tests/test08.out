5
invalid
5
5
